cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -706,6 +704,22 @@ func (s *Server) handleDebugList(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handleDebugMerge triggers a merge even if shard merging is not enabled. Users
+// can run this command during periods of low usage (evenings, weekends) to
+// trigger an initial merge run. In the steady-state, merges happen rarely, even
+// on busy instances, and users can rely on automatic merging instead.
+func (s *Server) handleDebugMerge(w http.ResponseWriter, _ *http.Request) {
+
+	// A merge operation can take very long, depending on the number merges and the
+	// target size of the compound shards. We run the merge in the background and
+	// return immediately to the user.
+	//
+	// We track the status of the merge with metricShardMergingRunning.
+	go func() {
+		s.doMerge()
```

Comment:
probably over engineering, but we could be writing the merge logs to w? Maybe better for a follow-up. Also I suppose the admin can just look at the logs of the pod rather.

So I'd say lets just write something to w like "merge enqueued"